packages/api/lib/granules.js

```diff
@@ -62,10 +64,37 @@ function getGranuleProductVolume(granuleFiles = []) {
     .reduce((x, y) => x + y, 0);
 }
 
+// TODO default arguments
```

Comment:
Address TODO - I'm assuming this is just adding default models. 

Nit: I'd suggest parameterizing this given the number of arguments.

Reply:
Yeah, definitely. Here's that commit:

https://github.com/nasa/cumulus/commit/7c6fb74b267334220afff90d9e7980d0cf5c9ab5

There is one weird thing. It works as-is but one of my defaults doesn't do what (I feel like) it should. See this comment:

https://github.com/nasa/cumulus/commit/7c6fb74b267334220afff90d9e7980d0cf5c9ab5#r46987116

Open to ideas. It has to be something small that I'm just missing. I'll keep at it but wanted to get this open for feedback.

Reply:
This has been fixed. There was a circular dependency created by having this function in `/lib/granules`. Moving it to a separate file solved the problem:

packages/api/lib/granule-delete.js